e2e/setup.ts

```diff
@@ -0,0 +1,49 @@
+import { chromium, firefox, webkit, type FullConfig } from '@playwright/test';
+import { CreateForkBody, createFork, forkRpcUrl } from './utils/tenderly';
+
+const forkConfig: CreateForkBody = {
```

Comment:
As we said, we probably need to fork from a specific block. Yes, using a dedicated wallet reduces the chance of unexpected results - but for every other needs (e.g. testing the explorer) you want to get the same data.

Reply:
The `forkConfig` has an params `blockNumber` to specify. I was waiting to discuss it with you during Sync. I should have left a comment about that sorry.

Reply:
As discussed - no other constraint - just pick what's current at the moment